client/dashboard/store-performance/index.js

```diff
@@ -201,7 +201,7 @@ export default compose(
 
 		if ( statKeys.length === 0 ) {
 			return {
-				hiddenBlocks,
+				hiddenIndicators: [],
```

Comment:
We are using `hiddenBlocks` everywhere else in the file to keep it consistent with other sections, is there any reason to change it here?

Reply:
Oops, that was a left over change from debugging. Fixed.